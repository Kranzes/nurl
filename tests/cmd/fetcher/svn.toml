args = ["svn://svn.mplayerhq.hu/mplayer/trunk", "30000"]
