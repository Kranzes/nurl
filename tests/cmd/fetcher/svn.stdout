fetchsvn {
  url = "svn://svn.mplayerhq.hu/mplayer/trunk";
  rev = "30000";
  sha256 = "sha256-gmiB9GR2VfzTzdIWe6kVCldaau6Cq0F9J0exFZChT1o=";
}